src/client/js/Widgets/BranchStatus/BranchStatusWidget.js

```diff
@@ -218,6 +233,26 @@ define([
         this._ddBranchStatus.clear();
         this._outOfSync = true;
         if (this._client.isConnected()) {
+            if (this._config.autoFollowBranchWhenOutOfSync && this._client.getCommitQueue().length === 1) {
+                $.notify({
+                    icon: 'fa fa-share-alt fa-rotate-90',
+                    message: 'Ooops! Looks like someone is chaning the model concurretly .. ' +
+                    'Your last change was dropped, check branch history for details.'
```

Comment:
Don't think the history is going to tell anything about the dropped changes, which would be the meaningful info in this case (so I would just let the user know that the change was dropped).

Reply:
The commit will be head-less, but still available if you check `Recent commits`.
(Not super useful, but just in case someone wrote a novel and hit save xD)

Reply:
True.